visitors/http_rest/spec_pair_visitor.go

```diff
@@ -30,7 +30,7 @@ type SpecPairVisitor interface {
 	// A utility function for visiting a set of arguments in a method request or
 	// response. This is needed since the arguments are store in maps whose keys
 	// do not reliably identify the arguments.
-	VisitMethodArgs(self interface{}, ctxt PairContext, vm PairVisitorManager, left, right map[string]*pb.Data) Cont
+	VisitMethodArgs(self interface{}, ctxt PairContext, vm PairVisitorManager, left map[string]*pb.Data, leftMethodMeta *pb.HTTPMethodMeta, right map[string]*pb.Data, rightMethodMeta *pb.HTTPMethodMeta) Cont
```

Comment:
Is it possible to get the metadata from the context, e.g. `ctxt.GetLeftOuterData().GetMeta().GetHTTPMeta()`?  (I know I'm getting the names of the methods there wrong, but hopefully you get the gist.)

Reply:
Done in f264555.